cdap-ui/app/features/tracker/services/my-tracker-api.js

```diff
@@ -29,7 +29,12 @@ function myTrackerApi(myCdapUrl, $resource, myAuth, myHelpers, UI_CONFIG) {
       timeSincePath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/auditmetrics/time-since',
       exploreQueryPath = '/namespaces/:namespace/data/explore/queries',
       baseQueryPath = '/data/explore/queries/:handle',
-      truthMeterPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tracker-meter';
+      truthMeterPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tracker-meter',
+      tagsPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tags',
+      demotePreferredTagsPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tags/demote',
+      promoteUserTagsPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tags/promote',
+      validateTagsPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tags/validate',
+      deletePreferredTagsPath = '/namespaces/:namespace/apps/' + UI_CONFIG.tracker.appId + '/services/' + UI_CONFIG.tracker.serviceId + '/methods/v1/tags/preferred';
```

Comment:
Can we clean up these paths, use the tagsPath base, and add whatever path you need